kuma/users/views.py

```diff
@@ -863,10 +863,7 @@ def create_stripe_subscription(request):
     try:
```

Comment:
This Django view is now obsolete, right? 
It's killing me that we have two Stripe integration forms. 
How about, instead, we take this PR to include a deletion of ALL things Stripe related on the Edit Profile page.

Reply:
Pretty sure Kadir wanted us to keep this view around, at least for a while

Reply:
Yeah. I suppose so. Just because of us trying to be so careful with the rollout. 

You know what could work... 
* File an issue to remove subscription form from the Edit Profile
* Have 3 different waffle flags.
  * One for the display of the banner (let's call/keep calling this `subscription_banner`)
  * One for the fundamentals such as Subscription management and `api.v1.views.create_subscription` (let's keep call this `subscription`)
  * One for the form being on the Subscription landing page (let's call this `subscription_form`)

This way, we get rid of the duplication and all its risks. We don't break anything for those extremely select few who have already signed up. And we can fine-tuning'ly control the features. 

All of the above **can wait, but mustn't wait long**. We mustn't let feature creep make us more fragile. Subscription is no longer an MVP. It's a "first-class citizen feature" in Kuma.

Reply:
Right yeah those three flags make sense to me. I would've thought we already have an issue for it, given the length at which we talked about it. I just copied your comment into a new issue for that one: #6782

Reply:
Conversation moved to https://github.com/mdn/kuma/issues/6783